composer.json

```diff
@@ -22,7 +22,7 @@
         "drupal/migrate_report": "~1.0-alpha1",
         "drupal/migrate_run": "~1.0-alpha1",
         "drupal/migrate_spreadsheet": "~1.0-beta1",
-        "drupal/og": "dev-1.x#34a9425ffeeb267ceeba21261f4719d0ced36e58",
+        "drupal/og": "dev-ISAICP-2709 as dev-1.x",
```

Comment:
I've created a custom route to the same controller, but with the 'administer members' permission.